packages/common/src/common.ts

```diff
@@ -690,6 +630,7 @@ export class Common {
         typeof blockOrTime === 'number' &&
         blockOrTime !== 0 &&
         blockOrTime !== prevBlockOrTime &&
+        // TODO: Decide if we should remove this since we no longer support TTD merge switch
```

Comment:
Remaining TODO

Reply:
This needs to stay in, same argument as the broader fork hash comment, will just remove the TODO.

Reply:
(so, to summarize: having the Paris check here is not about a TTD merge switch but a simple by-block-number merge switch + still needed for correct post-Merge fork hash calculation)